tinkerbell/hegel/values.yaml

```diff
@@ -0,0 +1,25 @@
+hegel: # a top-level key like this enables importing of all sub-keys in parent charts
+  deploy: true
+  env:
+  - name: TRUSTED_PROXIES
```

Comment:
Do we want this by default?

Reply:
no, i still need to figure a way out to maybe this automatic.

Reply:
this is now handled at `helm install` time. see the README.md

```bash
trusted_proxies=$(kubectl get nodes -o jsonpath='{.items[*].spec.podCIDR}' | tr ' ' ',')
helm install stack stack/ --create-namespace --namespace tink-system --wait --set "boots.boots.trustedProxies=${trusted_proxies}" --set "hegel.hegel.trustedProxies=${trusted_proxies}"
```